scripts/create_makefile.rb

```diff
@@ -96,7 +96,7 @@ def reject_mock_files(file)
     if not makefile_targets.include? module_obj
         makefile_targets.push(module_obj)
         mkfile.puts "#{module_obj}: #{module_src}"
-        mkfile.puts "\t${CC} -o $@ -c $< ${TEST_CFLAGS} -I #{SRC_DIR} ${INCLUDE_PATH}"
+        mkfile.puts "\t${CC} -o $@ -c $< ${TEST_CFLAGS} -I ${SRC_DIR} ${INCLUDE_PATH}"
         mkfile.puts ""
```

Comment:
We're puts-ing the SRC_DIR var to the new generated makefile so shouldn't we just use that var here?